### PR TITLE
Question and result pages

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Added question and result pages.
question page link: https://sharp-lalande-a4bd0e.netlify.app/quiz/question/question.html
result page link: https://sharp-lalande-a4bd0e.netlify.app/quiz/result/result.html

Screenshot 2022-02-20 at 8 25 37 AM
Screenshot 2022-02-20 at 8 25 53 AM

### CODE REVIEW

**Review by malaykhakhar — APPROVED** [review]
The code looks good and is pretty clean. You can merge it.

**Comment by prankurpandeyy** [comment]
looks good to merege

**Review by prankurpandeyy — APPROVED** [review]

File: quiz/question/question.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            <a class="navbar navbar-m-lr-1 navbar-m-l-2" href="#">
+                <i class="fa-brands fa-github"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-linkedin"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+        </div>
```
Comment (Ankur9669) [inline quiz/question/question.html:31]: fix: added links to the nav

File: quiz/question/question.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                <p>Question: 1 / 5</p>
+                <p>Score: 0</p>   
+            </div>
+            <h4 class="m-b-2rem">
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quod inventore sapiente repudiandae, quisquam culpa ut reprehenderit consectetur quos tenetur odio illo commodi magnam cumque voluptatibus, aperiam nesciunt, enim sint. Architecto.
+            </h4>
+            <button class="btn btn-primary">Answer 1</button>
+            <button class="btn btn-primary">Answer 2</button>
+            <button class="btn btn-primary">Answer 3</button>
+            <div class="flex">
```
Comment (Ankur9669) [inline quiz/question/question.html:49]: Suggestion: I think these buttons would be radio buttons

File: quiz/result/result.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            <h2 class="text-center m-bottom">Result</h2>
+            <h3 class="text-center m-bottom">Final Score: 10</h3>
+            <div class="m-bottom">
+                <h4>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quod inventore sapiente repudiandae, quisquam culpa ut reprehenderit consectetur quos tenetur odio illo commodi magnam cumque voluptatibus, aperiam nesciunt, enim sint. Architecto.
+                </h4>
+                <button class="button">Answer 1</button>
+                <button class="button wrong-answer">Answer 2</button>
+                <button class="button correct-answer">Answer 3</button>
+            </div>
```
Comment (Ankur9669) [inline quiz/result/result.html:47]: fix: These would not be buttons as here you are showing the result to the user, here user would not be clicking in these

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
UI improvement:
image
The prev and next buttons are very small

image


Keep some margins beetween each questions

File: quiz/question/question.html (hunk 1 above)
Comment (yaswanthmyneni) [reply quiz/question/question.html:31]: i have them in navbar too.

**Pull request merged**